tests/test_aug_db.py

```diff
@@ -0,0 +1,112 @@
+"""Tests for Rago package: Vector DBs."""
+
+import tempfile
+
+from functools import partial
+from typing import Generator, Optional
+
+import chromadb
+import pytest
+
+from chromadb.config import Settings
+from rago.augmented.db.chroma import ChromaDB
+
+API_MAP = {
+    # ChromaDB: 'api_key_openai',
+}
+
+dbs = [
+    partial(
+        ChromaDB,
+        **dict(),
+    ),
+]
+
+
+@pytest.fixture
+def temp_dir() -> Generator[str, None, None]:
+    """Provide temporary directory."""
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        yield tmp_dir
+
+
+def create_chroma_client(
+    persist_directory: Optional[str] = None,
+) -> chromadb.Client:
+    """Create a Chroma client instance with specified persist directory."""
+    settings = Settings()
+    if persist_directory:
+        settings = Settings(
+            persist_directory=persist_directory, is_persistent=True
+        )
+    return chromadb.Client(settings=settings)
+
+
+# def create_chroma_instance(
```

Comment:
I guess this code was commented out but it is not necessary anymore.
I will not block this pr to be merged, but it would be nice a follow up PR to remove unnecessary comments.